extension-manifest-v3/src/pages/panel/views/detailed.js

```diff
@@ -28,7 +28,7 @@ export default define({
       ${store.ready(stats) &&
       html`
         ${Object.keys(stats.byCategory)
-          .sort((a, b) => order.indexOf(a[0]) - order.indexOf(b[0]))
```

Comment:
looks like we had a bug in here

Reply:
I assume that added `sortCategories` function solves the problem, isn't it? ;)